### DISCUSSION TITLE
Image Magick blend 2 cropped/resize images together in the middle

### DISCUSSION THREAD

**Discussion by StealthRT** [dlemstra/Magick.NET#1112] [Help] [answered]
Hey all I have this code below that takes 2 images and merges them together with it fading in the center:

    convert testingl.jpg -gravity West ^
    testingr.jpg -gravity East ^
    blend_mask.png -extent 1080x440 -gravity center -composite bothBlended.jpg

The above produces this:

[![enter image description here][1]][1]

Using this mask:

[![enter image description here][2]][2]

Taken from these 2 images (lowered res to fit on here):

**testingl.jpg** (original size 1224 x 1632)

[![enter image description here][3]][3]

**testingr.jpg** (original size 828 x 1792)

[![enter image description here][4]][4]

This code works great as-is. Does what I need it to do but with one exception - I am wanting to get more of each image into it. Like resize the image, crop from the center the image then take that and blend it. I need to keep the same 1080 x 440 overall size.

Do that with both would look something like this:

[![enter image description here][5]][5]

  [1]: https://i.stack.imgur.com/L0bL6.jpg
  [2]: https://i.stack.imgur.com/L6MAm.png
  [3]: https://i.stack.imgur.com/6xShN.png
  [4]: https://i.stack.imgur.com/lKgkL.png
  [5]: https://i.stack.imgur.com/Pwkyp.jpg

**Comment by StealthRT** [comment]
Got this working.

**Reply by dlemstra** [reply]
Could you share how? That will probably help someone in the future.

**Comment by StealthRT** [comment, marked as answer]
```
convert ^
( testingl.jpg -resize 610x440^^ -gravity West -extent 1080x440 ) ^
( testingr.jpg -resize 610x440^^ -gravity East -extent 1080x440 ) ^
blend_mask.png -blur 0x7 ^
-composite bothImagesMerged.jpg
```
